apps/web/pages/availability/[schedule].tsx

```diff
@@ -212,7 +212,7 @@ export default function Availability() {
                 aria-label={t("delete")}
                 className="hidden sm:inline"
                 disabled={schedule?.isLastSchedule}
-                tooltip={t("requires_at_least_one_schedule")}
+                tooltip={t("delete")}
```

Comment:
don't change this tooltip

Reply:
> don't change this tooltip

Sure @Udit-takkar will update the PR shortly

Reply:
@Udit-takkar Updated the PR accordingly, please check